api/Document.json

```diff
@@ -3485,6 +3485,58 @@
           }
         }
       },
+      "fonts": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/fonts",
+          "support": {
+            "chrome": {
+              "version_added": "60"
+            },
+            "chrome_android": {
+              "version_added": "60"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "41",
+              "notes": "First implemented in Firefox 35 behind the flag <code>layout.css.font-loading-api.enabled</code>. Enabled by default in Firefox 41."
```

Comment:
This should be represented using the `flags` attribute instead: https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#flags

Like this:

```json
"firefox": [
  {
    "version_added": "41"
  },
  {
    "version_added": "35",
    "version_removed": "41",
    "flags": [
      {
        "type": "preference",
        "name": "layout.css.font-loading-api.enabled",
        "value_to_set": "true"
      }
    ]
  }
]
```

Reply:
Thanks for the pointer @connorshea. I've updated the PR.